### ISSUE TITLE
drop down windows don't open fully

### ISSUE THREAD

**Issue by WizardKate** [koditik/moodle-theme_boosted#2]
I'm having an issue with the drop down windows not opening fully in the custom reports area.
![error screen](https://github.com/koditik/moodle-theme_boosted/assets/3276205/0818b45f-046c-40ed-b174-dd1d39eaa99e)
![error screen 2](https://github.com/koditik/moodle-theme_boosted/assets/3276205/0b5df3a7-9167-426f-b9c8-eede30a312c3)

I've tried setting the size of the theme - it's at the default 80% now.
I am running  4.1.2+ (Build: 20230406)

**Comment by koditik** [comment]
Hi, thank you for contacting us. What Boosted theme version are you using?

**Comment by WizardKate** [comment]
Hi, thanks for getting back to me. I'm sorry I am not at my computer right now, but yesterday I downloaded the latest version that's available on Moodle and installed that , but still have the same issue. I uninstalled the version that i had before installing the lastest.Hope that helps!Kate On Jun 30, 2023 4:23 p.m., koditik ***@***.***> wrote:
Hi, thank you for contacting us. What Boosted theme version are you using?
